adapter_pipelines/ss2_single_sample/adapter_example_static.json

```diff
@@ -12,10 +12,10 @@
   "AdapterSmartSeq2SingleCell.stranded":"NONE",
   "AdapterSmartSeq2SingleCell.reference_bundle": "bf51d668-3e14-4843-9bc7-5d676fdf0e01",
   "AdapterSmartSeq2SingleCell.format_map": "gs://hca-dcp-mint-test-data/adapters/file_format_map.json",
-  "AdapterSmartSeq2SingleCell.dss_url": "https://dss.staging.data.humancellatlas.org/v1",
+  "AdapterSmartSeq2SingleCell.dss_url": "https://dss.dev.data.humancellatlas.org/v1",
```

Comment:
Our staging Lira's config is consuming this URL to connect to staging dss, would this change break the Lira?

Reply:
There is also another file called `adapter_example_static_integration.json` specifically for integration environments.

Reply:
Yes, I will undo this before merging. I was trying to get the integration test to pass, and only dev has v5 bundles so far.